### DISCUSSION TITLE
[Please Help]AssertionError: The observation space must be an image or dictionary observation space

### DISCUSSION THREAD

**Discussion by skgr07** [microsoft/AirSim#4864] [Support Q&A]
I cannot run the RL examples in Airsim because of this error. 
Is there anyone who had solved or who can help me with this error? 
I've googling on this issue for a week, but i can't find the solution.

Traceback (most recent call last):
  File "d:\AirSim\End-to-end_Autonomous_Driving_Car_Simulation-main\PythonClient_working\car\reinforcement_learning\rl_car.py", line 36, in <module>
    env = VecTransposeImage(env)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\AppData\Local\Programs\Python\Python311\Lib\site-packages\stable_baselines3\common\vec_env\vec_transpose.py", line 44, in __init__
    assert is_image_space(venv.observation_space) or isinstance(
AssertionError: The observation space must be an image or dictionary observation space
Exception ignored in: <function AirSimCarEnv.__del__ at 0x000001FD52A1A520>

**Comment by liangChengGit** [comment]
do you handle this problem ?
i have same problem

**Reply by lin77tar** [reply]
I've seen this before, so try lowering the stablebaseline version to 1.7.

**Reply by dang-kf** [reply]
![QQ截图20240117173440](https://github.com/microsoft/AirSim/assets/112178837/d1444308-fcb5-4296-b405-ba4c5c8ca439)
I've got this version down to 1.7.0, but still nothing

**Reply by Ritv1k** [reply]
what airsim version are you using does it work with airsim 1.8.1 I am now having  this error: "Function 'simGetImages' was called with an invalid number of arguments. Expected: 3, got: 2 "
